src/Components/Inventory.tsx

```diff
@@ -32,41 +27,18 @@ function Inventory({ProfileHandler,
                     ProgressHandler,
                     InventoryHandler,
                     ViewHandler,
-                    backToHomeHandler} : Props) {
+                    backToHomeHandler,
+                    SettingsHandler} : Props) {
     return (
         <div style={{ margin: 0, padding: 0, height: '100vh', position: 'relative' }}>
-            <div className="blue-half" style={{ width: '20%', height: '100%', backgroundColor: '#3127A0', position: 'fixed', left: 0}}>
-                <div id="website-title">Website.com</div>
-                <button id="sidebar-learning-progress-button" onClick={ProgressHandler}>
-                    <img src={learning_icon} alt="Learning icon"/>
-                    Learning Progress
-                </button>
-                <button id="sidebar-profile-button" onClick={ProfileHandler}>
-                    <img src={profile_icon} alt="Profile Icon"/>
-                    Profile
-                </button>
-                <button id="sidebar-help-center-button">
-                    <img src={help_center_icon} alt="Help Center Icon"/>
-                    Help Center
-                </button>
-                <button id="sidebar-setting-button">
-                    <img src={setting_icon} alt="Setting Icon"/>
-                    Setting
-                </button>
-                <button id="sidebar-theme-button">
-                    <img src={theme_icon} alt="Theme Icon"/>
-                    Theme
-                </button>
-                <button id="sidebar-selected-inventory-button">
-                    <img src={selected_inventory_icon} alt="Selected Inventory Icon"/>
-                    Inventory
-                </button>
-                <div id="separation-line"></div>
-                <button id="sidebar-logout-button" onClick={getStartedHandler}>
-                    <img src={logout_icon} alt="Logout Icon"/>
-                    Log out
-                </button>
-            </div>
+            <NavigationBar 
+                ProfileHandler={ProfileHandler}
```

Comment:
Do we need to keep these handlers as props? Or can we just declare them inside the component?

Reply:
i think the handlers call the setPage function which is defined in App.tsx so we have to defined these handlers in App.tsx as well, but maybe there's a way to bypass it idk

Reply:
This should be a common component that all pages use and they all should behave the same on clicking those tabs. I believe this should be kept inside the component

Reply:
OK, we can do that. But I think there'll be a lot to change (are you saying we should move a good chunk of the code in app.tsx into another separate file?), since hs said other groups' merging depends on the navbar fixes, I think we should merge them for now and make those changes in a later commit.

Reply:
Most of those handlers are just setting the route. I dont see any reason to leave it outside BlueNavBar